doc/compiler/ProblemDetermination.md

```diff
@@ -216,13 +216,54 @@ compilation altogether, using the `exclude=method` parameter:
 ### Identifying the failing Optimization.
 
 Most transformations that can be elided are guarded in the source code by a special
-check called `performTransformation` that can be controlled by the option
-`lastOptIndex=N`. This tells the compiler to perform `N` optional transformations,
-then proceed to only do required operations. In this sense, `lastOptIndex=-1` is an
-even stronger property than `optLevel=noOpt`.
+check called [performTransformation](https://github.com/eclipse/omr/blob/f5a790b68037be3a48b6e5c4f7e8ae6ad3e30f00/compiler/compile/OMRCompilation.hpp#L271-L274)
+([more details](https://github.com/eclipse/omr/blob/8f5212d52b533cdf9d6de9c2bd8a3e01e83e8b9d/compiler/compile/OMRCompilation.hpp#L170-L226)).
```

Comment:
It occurs to me that this doesn't mention the fact that [`performOptimization` might entirely skip on optimization based on the settings of `firstOptIndex` and `lastOptIndex`](https://github.com/eclipse/omr/blob/8f5212d52b533cdf9d6de9c2bd8a3e01e83e8b9d/compiler/optimizer/OMROptimizer.cpp#L1670-L1671) if it's not a required optimization.  In such cases, we'll never even get to the checks in `performTransformation`.

Reply:
Addressed in 75e009b